Gemfile.lock

```diff
@@ -78,15 +77,9 @@ GEM
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
-    rest-client (2.1.0-x64-mingw32)
```

Comment:
Is this removing the rest-client from the lock file?

Reply:
It's on line 75 now.